### DISCUSSION TITLE
Open file after creation does not work, I might be doing something wrong.

### DISCUSSION THREAD

**Discussion by fayazara** [nuxtrdev/nuxtr-vscode#80] [Questions]
Hey,

I love this extension, planning on using in my new company's project.

However, the Open file after creating does not work for me, either I read the docs wrong or it's not working for my setup, can you please help @adhamfarrag

As per the readme, it says to add these, I am assuming these are to be added in the `.vscode/settings.json` which I have.

```
Customize Vue file templates with these settings:

Additionally, you can choose to automatically open the newly created file with:

"nuxtr.openItemsAfterCreation": true
```

**Comment by adhamfarrag** [comment]
Hi @fayazara 

Thanks for your kind words and congratulations on the new job. :) 

Weird you're experiencing this. Just to make sure, did you refresh VSCode after applying the change? 


I'm not on my laptop but can you also try using the setting globally snd see if there's a difference?

**Reply by adhamfarrag** [reply]
@fayazara I couldn't reproduce even on a project configuration level.

**Comment by fayazara** [comment]
Let try to reproduce and send a screencast.

**Comment by fayazara** [comment]
That's odd, restarted my vscode and it seems to work now, sorry about that.

**Reply by adhamfarrag** [reply]
Nothing to be sorry about. I'll work on making VSCode apply changes without restart but low priority for now. 

 Keep building cool things. 💚
